SConstruct

```diff
@@ -349,6 +349,12 @@ def scons(): # pylint: disable=too-many-locals
 
     platform_arm = is_platform_arm()
 
+    if 'VIRTUAL_ENV' in os.environ:
+        env.PrependENVPath('PATH', f"{os.environ['VIRTUAL_ENV']}/bin")
```

Comment:
(pylint-no-member) Instance of 'DefaultEnvironment' has no 'PrependENVPath' member